### PR TITLE
Storage Capacity Notifications

### PR DESCRIPTION
**Pull request by iroelperez**

### CODE REVIEW

**Comment by iroelperez** [comment]
Note: the email notification part will be replaced later using the new system based on queues.

File: Portal/src/Datahub.Functions/AzureConfig.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    
+    #endregion
+
+    #region Storage Notifications
+
+    public string StorageQueueConnection => _config["DatahubStorageConnectionString"] ?? "";
+    public string MaxStorageCapacity => _config["MAX_STORAGE_CAPACITY"] ?? "180000000000"; //"2000000000000";
+    
+    #endregion
+}
```
Comment (yjmrobert) [inline Portal/src/Datahub.Functions/AzureConfig.cs:37]: I love this config class, this is what we need for the Datahub config options
cc @ErikApption

**Review by yjmrobert — APPROVED** [review]
Nice, thanks for cleaning up some of the utils/config along the way.

File: Portal/src/Datahub.Functions/AzureConfig.cs (hunk 1 above)
Comment (iroelperez) [reply Portal/src/Datahub.Functions/AzureConfig.cs:37]: The idea is to have all the config names in one place. Also validate some configurations, etc.

**Pull request merged**